### PR TITLE
andWhere and equals as default

### PR DESCRIPTION
**Pull request by nicosommi**

### CODE REVIEW

File: es5/spec/database/query/equalTo.spec.js [hunk 1]
```diff
@@ -86,6 +89,34 @@ describe(".equalTo(query)", function () {
     queryA.equalTo(queryB).should.be["false"];
   });
 
+  describe("(where)", function () {
```
Comment (dcrockwell) [inline es5/spec/database/query/equalTo.spec.js]: This is not the right file for this. we're not testing mocks in this file.

If anything, this should be `mock.where.spec.js`

File: es6/lib/query.js [hunk 2]
```diff
@@ -181,7 +181,11 @@ export default class Query {
 					const theirLink = theirChain[theirIndex];
 					const theirArguments = theirLink.options;
 
-					if (ourLink.name === theirLink.name) {
+					//so andWhere and where are treated both as the same
+					const theirLinkName = theirLink.name.replace("andWhere", "where");
```
Comment (dcrockwell) [inline es6/lib/query.js]: I'm not sure this is a good idea. We need to discuss this.

File: es6/lib/query.js [hunk 3]
```diff
@@ -206,6 +210,10 @@ export default class Query {
 	}
 
 	[addChain](chainName, options) {
+		const chainNameId = chainName.replace("andWhere", "where"); //treat where equals to andWhere
```
Comment (dcrockwell) [inline es6/lib/query.js]: replacing it again?

File: es6/spec/database/query/equalTo.spec.js [hunk 4]
```diff
@@ -4,10 +4,14 @@ const databaseConfig = require("../../../../database.json").testing;
 
 describe(".equalTo(query)", () => {
 
-  let database;
+  let database,
+    mockUsers;
 
   beforeEach(() => {
 		database = new Database(databaseConfig);
+    mockUsers = [
```
Comment (dcrockwell) [inline es6/spec/database/query/equalTo.spec.js]: tabbing

File: es6/spec/database/query/equalTo.spec.js [hunk 5]
```diff
@@ -80,6 +85,30 @@ describe(".equalTo(query)", () => {
     queryA.equalTo(queryB).should.be.false;
   });
 
+  describe("(where)", () => {
+    it("should treat where equals to andWhere", done => {
```
Comment (dcrockwell) [inline es6/spec/database/query/equalTo.spec.js]: this is not a good test name

File: es6/spec/database/query/equalTo.spec.js [hunk 6]
```diff
@@ -80,6 +85,30 @@ describe(".equalTo(query)", () => {
...
+      const name = "aName";
+
+      database.mock.select(/.*/).from(/u.*rs/).where("created_at", ">", /.*/).where("name", name).results(mockUsers);
```
Comment (dcrockwell) [inline es6/spec/database/query/equalTo.spec.js]: If we're just testing .where, we don't really want to have all of this complicated regex. If regex isn't necessary to exemplify; simplify instead.

File: es6/spec/database/query/equalTo.spec.js [hunk 7]
```diff
@@ -80,6 +85,30 @@ describe(".equalTo(query)", () => {
     queryA.equalTo(queryB).should.be.false;
   });
 
+  describe("(where)", () => {
```
Comment (dcrockwell) [inline es6/spec/database/query/equalTo.spec.js]: This whole block should be in `mock.where.spec.js`

File: es6/spec/database/mock.where.spec.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
+import Database from "../../lib/database.js";
+
+const databaseConfig = require("../../../database.json").testing;
+
+describe(".where", () => {
```
Comment (dcrockwell) [inline es6/spec/database/mock.where.spec.js]: .mock.where(...options)

File: es6/spec/database/mock.where.spec.js [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+		database.mock.select("*").from("users").where("created_at", ">", /.*/).where("name", name).results(mockUsers);
+
+		database.select("*").from("users").where("created_at", ">", "2015-10-02 21:39:14").andWhere("name", name).results((error, rows) => {
+			if (error) { throw error; }
+			rows.should.eql(mockUsers);
+			done();
+		});
+	});
+
+	it("should use equals as the default operator", done => {
```
Comment (dcrockwell) [inline es6/spec/database/mock.where.spec.js]: Isn't this the same test as the one before?

Comment (nicosommi) [inline es6/spec/database/mock.where.spec.js]: no, this is testing that it works with equals as default, while the other is testing that andWhere and where are treated as equals

Comment (dcrockwell) [inline es6/spec/database/mock.where.spec.js]: Gotcha. I see, now. Thanks

**Pull request merged**